src/fragments/lib/devpreview/ios/getting_started/10_intro.mdx

```diff
@@ -0,0 +1,7 @@
+The Developer Preview of the Amplify iOS Library is now exclusively using Swift and provides developers the ability to add cloud-based Auth, Storage, Data, and APIs to their apps. With this version, Swift developers will be able to debug and contribute to the underlying open-source codebase completely in Swift. We plan to incrementally add more Amplify features including Swift-based language features like structured concurrency.
```

Comment:
> ... ability to add cloud-based Auth, Storage, Data, and APIs to their apps.

Is it accurate that cloud-based Data and APIs can be added exclusively using Amplify iOS Developer Preview in combination with the Swift SDK?